docs/mendel.md

```diff
@@ -0,0 +1,30 @@
+# nf-core/configs: MENDEL Configuration
+
+All nf-core pipelines have been successfully configured for use on the MENDEL CLUSTER at the Gregor Mendel Institute (GMI).
+
+To use, run the pipeline with `-profile conda,mendel`. This will download and launch the [`mendel.config`](../conf/mendel.config) which has been pre-configured with a setup suitable for the MENDEL cluster. A Conda environment will be created automatically and software dependencies will be downloaded from ['bioconda'](https://bioconda.github.io/).
+
+Theoretically, using `-profile singularity,mendel` would download a docker image containing all of the required software, and convert it to a Singularity image before execution of the pipeline. However, there is a regression in the Singularity deployment on MENDEL which renders containers downloaded from public repositories unusable because they lack the /lustre mountpoint.
+
+If you want to run the pipeline containerized anyway you will have to build the image yourself (on a machine where you have root access) using the provided `Singularity` file in the pipeline repository:
```

Comment:
Note that we are systematically removing all of these files from pipelines now for simplicity (they were originally added for use with singularity-hub, which we are now not pursuing).

Reply:
That is unfortunate. This means I probably should remove the singularity parts entirely from the config and documentation and just support conda

Reply:
Nope, you misunderstand me. We're not dropping support for singularity (most people involved in the project use it as their primary environment), we're dropping support for singularity _hub_ - that is, the singularity container registry: https://singularity-hub.org/

We still have support for using singularity, we just get the containers by building from dockerhub instead. We only ever put a couple of pipelines on singularity hub and it never worked properly.

Reply:
I thought there was a misunderstanding too but it seems like the Docker -> Singularity conversion doesnt work for @phue . The Singularity images were being built manually from the Singularity file which will be taken out of most pipelines.

Reply:
Exactly, pulling Images from the registry doesn't work for us because our Kernel lacks overlayfs (and I reconfirmed with sysadmins, they won't fix it because the current cluster will go out of service soonish) . I have to build the containers manually. I really think t's best to just remove singularity support here until the new cluster is running and rely in conda in the meantime

Reply:
Oh sorry, my bad.

I think there _should_ be a way to build “child” containers where overlayfs isn’t an option. I feel like I remember thinking about this a while ago.

Basically have a mini recipe where you use `FROM` (or the singularity equivalent) to point to the nf-core image, then have a bunch of `mkdir` commands. Does that make sense?

Reply:
>  Does that make sense?

Not sure if I'm understanding this correctly. Wouldn't this also require some sort of recipe which again needs to be built manually?

The only way I see this working currently is either a) to make the docker image somehow writeable (not sure if that's even possible) and then use some sort of hack to add the missing mountpoint using something like `singularity.runOptions = 'mkdir /lustre &&'` or b) to change the dockerfiles to create the folder at buildtime.

But both would require global changes just to accommodate a bug in our institutes' infrastructure. I don't think we should start doing such things, right?

Reply:
I've never been able to test solutions to this as we don't have access to a server with overlayfs turned off. But I think that this one-off manual step for the singularity containers should work?

```bash
singularity pull --name nf-core-rnaseq-1.1.simg docker://nfcore/rnaseq:1.1
sudo singularity exec --writable nf-core-rnaseq-1.1.simg mkdir /lustre
```

See http://singularity.lbl.gov/archive/docs/v2-3/docs-changing-containers

Or my child container suggestion was to make your own singularity build script for each pipeline which would look like this:

```Singularity
Bootstrap: docker
From: nfcore/rnaseq:1.1

%post
    mkdir /lustre
```

I have a vague memory of @remiolsen having some kind of nicer run-time nextflow config option for binding directories in Singularity for non-overlayfs people too. But I can't remember the details, or whether it worked. Basically in your nextflow config file having this I think:
```nextflow
singularity.engineOptions = '-B /lustre:/lustre'
```